src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -628,6 +634,355 @@ public bool Verify()
             }
         }
 
+        /// <summary>
+        /// Creates a new X509 certificate from the file contents of an RFC 7468 PEM-encoded
+        /// certificate and private key.
+        /// </summary>
+        /// <param name="certPemFilePath">The path for the PEM-encoded X509 certificate.</param>
+        /// <param name="keyPemFilePath">
+        /// If specified, the path for the PEM-encoded private key.
+        /// If unspecified, the <paramref name="certPemFilePath" /> file will be used to load
+        /// the private key.
+        /// </param>
+        /// <returns>A new certificate with the private key.</returns>
+        /// <exception cref="ArgumentException">
```

Comment:
I think you missed changing this one to CryptographicException in the docs.

```suggestion
        /// <exception cref="CryptographicException">
```

Reply:
Ah, yeah I merged the ArgumentException and the CryptographicException but kept the wrong type name.